### ISSUE TITLE
Compannion dssappeared

### ISSUE THREAD

**Issue by P14S0P** [MarkusBordihn/BOs-Player-Companions#30]
**Describe the bug**
Well i made the special bone you need to get a corgi, it was a black one, i gave him the bone and the corgi disappeared.

**To Reproduce**
1. Get the Bone(Taming Object)
2. Give it to a black corgi.

**Expected behavior**
To get a new companion :C

**Screenshots**

**Debugging**
Before reporting a issue please check if there is anything relevant in the crash-reports, debug.log and latest.log file.

**Is the issue happening in combination with other mods? (If yes, please provide a full list)**

[latest.log](https://github.com/MarkusBordihn/BOs-Player-Companions/files/13402709/latest.log)

**Additional context**

**Comment by P14S0P** [comment]
I don't have it on my inventory also.

**Comment by MarkusBordihn** [comment]
Thanks for the report, according the log the mod has placed the item in the inventory:
```
Register companion WelshCorgi['Jessie'/9506, l='ServerLevel[??lol?nbl?o?n?]', x=-431.42, y=63.00, z=669.08, tamed=true, owner='51cb0917-8c5d-4acb-b457-149696e76c47'] for ServerPlayer['PIASOP'/183, l='ServerLevel[??lol?nbl?o?n?]', x=-432.53, y=64.00, z=670.09] ...
Gave player ServerPlayer['PIASOP'/183, l='ServerLevel[??lol?nbl?o?n?]', x=-432.53, y=64.00, z=670.09] captured companion item with welsh_corgi_black.
```

Please check if you could use the `/player_companions summon` command to summon your companion.
Normally such issues happen with modified clients or if you are using inventory mods which are not able to handle the tame items correctly (e.g. auto-sorting).

The mod itself double checks if the item is in the inventory and drops the item to the ground if it is not in the inventory.
See: https://github.com/MarkusBordihn/BOs-Player-Companions/blob/c764e74224664fc19f489c80ce44fd46dbeab6ba/src/main/java/de/markusbordihn/playercompanions/item/CompanionTameItem.java#L93

I will add an additional check to log the stored item position within the inventory to get more insides into this issue.

**Comment by MarkusBordihn** [comment]
I think I found the issue and released a fix with version 6.8.0.

Please let me know if this version fix the issue for you by putting the captured companion item in your inventory or by dropping the captured companion item to the ground (in the case the inventory is not available or full).

**Comment by P14S0P** [comment]
Ok i downloaded the version and now i have the campturer companion in the inventory <3 i can drop them and grab them. tyvm for the help!
